index.bs

```diff
@@ -5523,6 +5558,116 @@ opened</dfn> steps given |window|, |type|, |message|, and optional |default valu
 
 </div>
 
+## The emulation Module ## {#module-emulation}
+
+The <dfn export for=modules>emulation</dfn> module contains commands and events
+relating to emulation of browser APIs.
+
+### Definition ### {#module-emulation-definition}
+
+[=remote end definition=]
+
+<pre class="cddl remote-cddl">
+EmulationCommand = (
+  emulation.setGeolocationOverride
```

Comment:
In CDP there is also `clearGeolocationOverride`. I wonder if we should separate it as well.

Reply:
We could, but we also use `null` to reset the overrides in `browsingContext.setViewport` command.

Reply:
Good point. I'm fine with a single API then. But lets keep the comment open in case others have some feedback as well.

Reply:
If I'm reading Geolocation specification PR correctly, having `coordinates: null` makes the geolocation not available, instead of removing the emulation https://pr-preview.s3.amazonaws.com/w3c/geolocation/pull/170.html#dfn-set-the-system-coordinates:

> If coordinates is not given, clear the system location and thus make the position unavailable.

Reply:
In the [new geolocation PR](https://github.com/w3c/geolocation/pull/183/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R813), `null` is considered as "disable navigation and provide the real data".